components/amorphic/lib/listen.js

```diff
@@ -113,13 +123,28 @@ function listen(appDirectory, sessionStore, preSessionInject, postSessionInject,
 			)
 		)
 		.then(function logStart() {
-			logMessage('Amorphic has been started with versions: ');
+			let msg = 'Amorphic has been started with versions: ';
 			for (let packageVer in packageVersions) {
-				logMessage(packageVer + ': ' + packageVersions[packageVer]);
+				msg += packageVer + ': ' + packageVersions[packageVer] + ', ';
 			}
+			SupertypeSession.logger.info({
+				module: moduleName,
+				function: functionName,
+				category: 'request',
+				message: msg,
+				data: {
+					packageVersions: packageVersions
+				}
+			});
 		})
 		.catch(function error(e) {
-			logMessage(e.message + ' ' + e.stack);
+			SupertypeSession.logger.error({
+				module: moduleName,
+				function: functionName,
+				category: 'request',
+				message: e.message + ' ' + e.stack,
+				error: e
```

Comment:
error needs isHumanRelated required field. 
Since all error `isHumanRelated` field needs to be false ( in our case), we could at the time of listen create a child logger from passed logger and set the default for isHumanRelated. 

this.logger = logger.childLogger? logger.childLogger({error: {isHumanRelated: false}}) : logger.child({error: {isHumanRelated: false}})